gems/ruby-saml/CVE-2016-5697.yml

```diff
@@ -0,0 +1,10 @@
+---
+gem: ruby-saml
+framework: rails
+cve: 2016-5697
+url: https://github.com/onelogin/ruby-saml
```

Comment:
Can you provide a specific link that concerns this vulnerability? Perhaps https://github.com/onelogin/ruby-saml/commit/a571f52171e6bfd87db59822d1d9e8c38fb3b995 or some release?